### ISSUE TITLE
Editing the editing keys

### ISSUE THREAD

**Issue by gargargarrick** [IgnorantGuru/spacefm#721]
I would like to delete the focused path bar's Shift+Backspace keybinding. However, there is no menu item for "clear the path bar", so I cannot use Design Mode to alter its shortcut. Binding other items to Shift+Backspace to "override" it will not work because the editing keys take priority over other combinations, and it seems there is no entry for this behavior in the session file either.

Can the editing key combinations be customized in any way? Of course it it is fine if they cannot (and thanks for creating such a stupendous file manager in any case).

**Comment by IgnorantGuru** [comment]
No, those editing keys cannot be changed, short of forking the source and editing it.  It's fairly easy to maintain a fork on github though.

However, when I assign Shift+Backspace to a menu item, it does override the use of that key combo in the path bar.  I am using GTK2, which may differ from GTK3.
